ripley/lib/canvas_site_utils.py

```diff
@@ -327,52 +322,50 @@ def provision_course_site(uid, site_name, site_abbreviation, term_slug, section_
         course.update(course={'default_view': 'feed'})
 
     # Section definitions
-    section_feeds = []
+    canvas_section_payload = []
     section_roles = {}
-    explicit_sections_for_instructor = []
     all_sections = []
+    # Roles
+    account_roles = {}
+    for role_type in ['StudentEnrollment', 'TaEnrollment', 'TeacherEnrollment']:
+        account_roles[role_type] = next((r for r in list(account.get_roles()) if r.base_role_type == role_type), None)
```

Comment:
The fix is here and in ` _enroll_user_in_canvas_section`, where `account_roles` is used.